metadata.yaml

```diff
@@ -44,6 +44,9 @@ requires:
   spark-service-account:
     interface: spark_service_account
     limit: 1
+  zookeeper:
+    interface: zookeeper
```

Comment:
**nitpick:** I think this is `zookeeper_client`

Reply:
I took reference from the Kafka code base: https://github.com/canonical/kafka-k8s-operator/blob/main/metadata.yaml#L57.

Reply:
Nope, I believe we kept `zookeeper` since that was the initial name in Kafka, and changing the interface name would have been an huge pain (not backward compatible). Because of this, even the name in the [interface repository](https://github.com/canonical/charm-relation-interfaces/pull/144)